source/documentation/rules-e.html.md.erb

```diff
@@ -0,0 +1,218 @@
+
+##E1105
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt;<br/>THEN <br/>/*/Consignment/HouseConsignment/ConsignmentItem/Packaging/shippingMarks<br/><br/>format shall be set to an..42
+
+##E1107
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt;<br/>THEN <br/>/*/Consignment/HouseConsignment/ConsignmentItem/Commodity/descriptionOfGoods<br/><br/>format shall be set to an..280
+
+##E1109
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt;<br/>THEN<br/>/*/Consignment/grossMass AND<br/>/*/Consignment/HouseConsignment/ConsignmentItem/Commodity/GoodsMeasure/grossMass AND<br/>/*/Consignment/HouseConsignment/ConsignmentItem/ Commodity/GoodsMeasure/netMass<br/>format shall be set to n..11,3
+
+##E1111
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt;<br/>THEN <br/>/*/Consignment/HouseConsignment/ConsignmentItem/Packaging/numberOfPackages<br/><br/>format shall be set to n..5
+
+##E1114
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt;<br/>THEN<br/>/*/Consignment/PlaceOfLoading/location <br/><br/>format shall be set to an..17
+
+##E1115
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt; <br/>THEN <br/>/*/Enquiry/text AND <br/>/*/Recovery/text AND <br/>/*/ResponseInformation/text AND <br/>/*/RequestedInformation/text AND <br/>/*/FunctionalError/originalAttributeValue <br/>format shall be set to an..140 
+
+##E1116
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt;<br/>THEN<br/>/*/TransitOperation/cancelEnquiryNotificationText AND<br/>/*/RiskAnalysisIdentification/RiskAnalysis/RiskAnalysisResult/text <br/>format shall be set to an...350<br/><br/>
+
+##E1117
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt;<br/>THEN<br/>/*/Consignment/HouseConsignment/ConsignmentItem/PreviousDocument/complementOfInformation<br/>/*/Consignment/HouseConsignment/ConsignmentItem/SupportingDocument/complementOfInformation<br/><br/>format shall be set to an..26.<br/>
+
+##E1118
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt;<br/>THEN<br/>/*/Guarantee/GuaranteeReference/accessCode<br/><br/>format shall be set to an4<br/><br/>
+
```

Comment:
Should be `an4` be `an..4`?

Reply:
an4 = 4 characters exactly
an..4 = up to 4 characters.  
I think an4 is right